pages/index.vue

```diff
@@ -1,11 +1,188 @@
+        >
+          Play
+        </NuxtLink>
+        <img
+          src="~\assets\img\icons\light\homepage-arrow-axolotl-light.svg"
+          class="h-36 dark:hidden"
+        />
+        <img
+          src="~\assets\img\icons\dark\homepage-arrow-axolotl-dark.svg"
+          class="hidden h-36 dark:block"
+        />
+      </div>
+      <div class="row mb-48 mt-4 flex place-content-around font-heading">
+        <p class="w-96">accessibility</p>
+        <p class="w-96 text-axolotl-dark dark:text-axolotl-light">axolotl</p>
+      </div>
+      <div
+        class="mx-52 border-b-4 border-solid border-blue-4 p-5 text-3xl leading-tight"
+      >
+        <p class="text-4xl font-bold">What are we doing?</p>
+        <p class="text-justify indent-6">
+          Our Project aims to teach society how important web-accessibility is.
+          We want to show what an accessible website means to disabled people
+          and how it can improve their experience. Our mascot, axolotl, is
+          helping users to understand the importance of implementing
+          accessibility features.
+        </p>
+        <div class="my-8 flex justify-center">
+          <img src="~\assets\img\icons\accessolotl-logo.svg" />
+        </div>
+        <p class="text-4xl font-bold">What is WCAG?</p>
+        <p class="text-justify indent-6">
+          We follow the Web Content Accessibility Guidelines (WCAG) 2.1 to make
+          content for our website more accessible to a wider range of people
+          with disabilities. Additionally, we strive to meet Level A conformance
+          for our own website.
+        </p>
+        <p>
+          <NuxtLink
+            to="https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1"
+            target="_blank"
+            class="underline decoration-solid underline-offset-4"
+            >Get more information here!</NuxtLink
+          >
+        </p>
+      </div>
+      <div class="my-16 text-4xl font-bold" id="about">
+        <p>About us</p>
+      </div>
+      <div class="flex place-content-evenly text-2xl">
+        <div>
+          <div
+            class="mb-4 h-[250px] w-[250px] rounded-full bg-[url('~/assets/img/teammitglied_marco.jpg')] bg-cover"
+          ></div>
+          <p class="font-semibold">Marco Janderka</p>
+          <p>Product Owner</p>
+        </div>
+        <div>
+          <div
+            class="mb-4 h-[250px] w-[250px] rounded-full bg-[url('~/assets/img/teammitglied_lisa.jpg')] bg-cover"
+          ></div>
+          <p class="font-semibold">Lisa-Marie Hörmann</p>
+          <p>Scrum Master</p>
+        </div>
+        <div>
+          <div
+            class="mb-4 h-[250px] w-[250px] rounded-full bg-[url('~/assets/img/teammitglied_felix.jpg')] bg-cover"
+          ></div>
+          <p class="font-semibold">Felix Wollmann</p>
+          <p>Projektmitglied</p>
```

Comment:
Warum ist das auf Deutsch und nicht wie die restliche Seite auf Englisch?
```suggestion
          <p>Team member</p>
```